atm.js

```diff
@@ -1,4 +1,65 @@
-var checkingBalance = document.querySelector("div#checking_balance");
+function ATM(){
+  this.els = {
+    checkingBalance: $(".balance").eq(0),
+    checkingInput: $("input").eq(0),
+    button: $(".checkingButton")
+  }
+}
+
+ATM.prototype.calculateTotal = function(input){
+  var input = input;
+  console.log(input);
+
+//console.log(input + "this is my input");
+  //for (var i =0; i<this.balance.length; i++){
+  //console.log("this is my balance " + total);
+//}
```

Comment:
I like to see instrumentation, for debugging.  However, this one confuses me.  I don't see anything that has a "balance" method and the loop doesn't use the "i" var that you are incrementing.